src/app/services/locales/en-us/index.yaml

```diff
@@ -23,14 +23,27 @@ services:
       config: 'YAML'
     items:
       title: Services
-      intro: !!text/markdown |
-        A Service represents an application or microservice that is registered with the mesh and can be managed, monitored and secured through the mesh's policies.
       navigation:
-        mesh-service-list-view: MeshService
-        mesh-multi-zone-service-list-view: MeshMultiZoneService
-        mesh-external-service-list-view: MeshExternalService
-        service-list-view: Internal
-        external-service-list-view: External
+        mesh-service-list-view:
+          label: MeshService
+          description: !!text/markdown |
+            A <a href="https://kuma.io/docs/latest/networking/meshservice/#meshservice" target="_blank">MeshService</a> represents a destination for traffic from elsewhere in the mesh and can define several networking details. The behaviour of this resource depends on the zone type. 
+        mesh-multi-zone-service-list-view:
+          label: MeshMultiZoneService
+          description: !!text/markdown |
+            A <a href="https://kuma.io/docs/latest/networking/meshmultizoneservice/#meshmultizoneservice" target="_blank">MeshMultiZoneService</a> represents a group of `MeshService` resources in a loadbalanced multizone deployment. `MeshService` resources that are deployed across several clusters can be grouped by a `MeshMultiZoneService`.
+        mesh-external-service-list-view:
+          label: MeshExternalService
+          description: !!text/markdown |
+            **Experimental**: A <a href="https://kuma.io/docs/latest/networking/meshexternalservice/#meshexternalservice" target="_blank">MeshExternalService</a> is a policy that allows an application or microservice to interact with explicit destinations that are not part of the mesh.
```

Comment:
How come this one is marked as experimental and not the others?

Reply:
It's marked as experimental in the docs, but probably we should not do this here because we'd need to put an extra eye to keep it in sync? 🙂

Reply:
Yeah I think you're right, lets take it out of here 👍

Reply:
I'll remove experimental to reduce our efforts and keep it consistent. We can add it back if really necessary 🙂